docs/migration.adoc

```diff
@@ -139,7 +175,7 @@ The idea is to have a central backup server where multiple clients can connect t
 Because all the clients can't migrate at the same time, it must be made sure that the server is able to support multiple versions of rdiff-backup at the same time.
 
 ____
-*NOTE:* the same approach can be used to support multiple clients of 	different versions but the use case doesn't seem as useful, hence 	it is left to the interpretation of the reader.
+*NOTE:* the same approach can be used to support multiple clients of different versions but the use case doesn't seem as useful, hence it is left to the interpretation of the reader.
```

Comment:
Sorry, I know I can be a pain, but you've forgotten to remove the asterisks (and also the 4 underscores before and after, I realize).

Reply:
Done, I forgot to look at the other file!!!